### DISCUSSION TITLE
兄弟们，github可以改语言吗🤔，全是英文看着好难受啊

### DISCUSSION THREAD

**Discussion by ssj20051003** [share121/inter-knot#723] [其他]
兄弟们，github可以改语言吗🤔，全是英文看着好难受啊

**Comment by AusertDream** [comment]
语言改不了，但是浏览器有插件，可以帮你翻译
比如[这个插件](https://chromewebstore.google.com/detail/immersive-translate-trans/bpoadfkcbjbfhfodiogcnhhhpibjhbnh)

**Comment by justafish8787** [comment]
我直接按右键翻译:D

**Comment by zHysie** [comment]
[这个汉化插件挺好的](https://github.com/maboloshi/github-chinese)

**Comment by Xmug** [comment]
可以用[这个脚本](https://greasyfork.org/zh-CN/scripts/407485-github-internationalization)
